client/src/components/musicPlayer.jsx

```diff
@@ -1,24 +1,178 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import $ from 'jquery';
-import songs from '../../../sampleData.js';
+import styled from 'styled-components';
+
+
+const MusicPlayerContainer = styled.div`
+  display: flex;
+  flex-direction: row; 
+  justify-content: flex-end;
+  background: linear-gradient(135deg, #4D586E, #3A67C0);
+  height: 340px;
+  width: 1200px;
+  margin: 0 auto;
+  color: white;
+  padding-top: 15px;
+  padding-left: 25px;
+  padding-bottom: 15px;
+  padding-right: 15px;
+`;
+
+const ButtonTitleContainer = styled.div`
+  padding-top: 20px;
+  display: flex;
+  flex-direction: row; 
+`;
+
+const TitleArtistContainer = styled.div`
+  padding-left: 10px;
+  display: flex;
+  flex-direction: column; 
+  width: 400px;
+  font-weight: 100;
+`;
+
+const PlayButtonContainer = styled.div`
+  justify-content: flex-start;
+  border-radius: 50%;
+  border-color: #FF5836;
+  background: #FF5836;
+  width: 60px;
+  height: 60px;
+  cursor: pointer;
+  &:hover {
+    background: #FF4137;
+  }
+`;
+const PlayButton = styled.img`
+  margin-left: 7px;
+  margin-top: 4px;
+  width: 50px;
+  height: 50px;
+`;
+
+const ArtistContainer = styled.div`
+  
+`;
+
+const AlbumContainer = styled.div`
+  
+`;
+
+const AlbumArtContainer = styled.div`
+  width: 340px;
+  height: 340px;
+`;
+
+const GenreCreatedContainer = styled.div`
+  margin-left: auto;
+  margin-right: 20px;
+  flex-direction: column;
+  font-size: 16px;
+`;
+
+const CreatedAt = styled.div`
+  margin-top: 16px;
+  margin-bottom: 16px;
+`;
+
+const GenreContainer = styled.div`
+  flex-direction: row;
+  text-align: right;
+`;
+
+const Artist = styled.span`
+  background-color: black;
+  font-size: 16px;
+  color: #ccc;
+  padding-top: 2px;
+  padding-right: 7px;
+  padding-bottom: 3px;
+  padding-left: 7px;
+  cursor: pointer;
+  ${ArtistContainer}:hover &{
+    color: white;
+  }
+`;
+
+const Album = styled.span`
+  background-color: black;
+  font-size: 16px;
+  color: #ccc;
+  line-height: 1;
+  padding-top: 2px;
+  padding-right: 7px;
+  padding-bottom: 3px;
+  padding-left: 7px;
+  cursor: pointer;
+  ${AlbumContainer}:hover &{
+    color: white;
+  }
+`;
+
+const Genre = styled.span`
+  background: #999;
+  color: #fff;
+  cursor: pointer;
+  border-radius:20px;
+  border-top-color: rgb(153, 153, 153);
```

Comment:
is there a reason you have the four borders separately? if they're all the same you can just do:
 border: 1px solid rgb(...) 
to handle all 4 at once

Reply:
Ahh, this part I got lazy and copy pasted the stuff from inspecting the page to get it as close as I can. I might tweak it later on to clean it up, but it's low priority.